_events/2017-04-09-this-bridge-called-my-back.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: this-bridge-called-my-back
```

Comment:
Oh I forgot to reply to the question of quotes. Since identifiers and slugs should never have spaces or special characters in them I left the quotes out. We usually only need quotes for user formatted strings.